src/components/Map.js

```diff
@@ -48,17 +48,10 @@ const Map = withScriptjs(withGoogleMap(props => (
             id={org.id}
             organization={org}
             open={org.isMarkerOpen}
-          />
+          /> 
         ) }
 
-        () => {
 
-        props.organizations.filter(org => org.coordinates).map(org =>
-            <InfoWindowComponent
```

Comment:
The import for this InfoWindowComponent is giving an error. Since you removed the usage of this component, removing the import above resolve the error.